index.d.ts

```diff
@@ -1,146 +1,148 @@
+    * computed delay in milliseconds
+    */
+  computed: number,
+  raw: number | EventLoopDelayHistogram
+}
 
-  interface GCAggregatedEntry extends GCOpStats {
-    flags?: Map<GCFlag, GCOpStats>
-  }
+declare enum GCFlag {
+  // perf_hooks.constants.NODE_PERFORMANCE_GC_FLAGS_NO
+  no = 'no', // eslint-disable-line no-unused-vars
+  // perf_hooks.constants.NODE_PERFORMANCE_GC_FLAGS_CONSTRUCT_RETAINED
+  constructRetained = 'constructRetained', // eslint-disable-line no-unused-vars
+  // perf_hooks.constants.NODE_PERFORMANCE_GC_FLAGS_FORCED
+  forced = 'forced', // eslint-disable-line no-unused-vars
+  // perf_hooks.constants.NODE_PERFORMANCE_GC_FLAGS_SYNCHRONOUS_PHANTOM_PROCESSING
+  synchronousPhantomProcessing = 'synchronousPhantomProcessing', // eslint-disable-line no-unused-vars
+  // perf_hooks.constants.NODE_PERFORMANCE_GC_FLAGS_ALL_AVAILABLE_GARBAGE
+  allAvailableGarbage = 'allAvailableGarbage', // eslint-disable-line no-unused-vars
+  // perf_hooks.constants.NODE_PERFORMANCE_GC_FLAGS_ALL_EXTERNAL_MEMORY
+  allExternalMemory = 'allExternalMemory', // eslint-disable-line no-unused-vars
+  // perf_hooks.constants.NODE_PERFORMANCE_GC_FLAGS_SCHEDULE_IDLE
+  scheduleIdle = 'scheduleIdle' // eslint-disable-line no-unused-vars
+}
+declare interface GCOpStats {
+  count: number,
+  total: number,
+  average: number
+}
+declare interface GCAggregatedEntry extends GCOpStats {
+  flags?: Map<GCFlag, GCOpStats>
+}
+declare interface GCMetric {
+  // perf_hooks.constants.NODE_PERFORMANCE_GC_MAJOR
+  major: GCAggregatedEntry,
+  // perf_hooks.constants.NODE_PERFORMANCE_GC_MINOR
+  minor: GCAggregatedEntry,
+  // perf_hooks.constants.NODE_PERFORMANCE_GC_INCREMENTAL
+  incremental: GCAggregatedEntry,
+  // perf_hooks.constants.NODE_PERFORMANCE_GC_WEAKCB
+  weakCb: GCAggregatedEntry
+}
 
-  interface DocData {
-    /**
-     * Event loop delay (ms).
-     */
-    eventLoopDelay: number,
-    /**
-     * Object containing memory usage stats.
-     */
-    memory: {
-      /**
-       * RSS memory (bytes).
-       */
-      rss: number,
-      /**
-       * Total heap Memory (bytes).
-       */
-      heapTotal: number,
-      /**
-       * Heap memory used (bytes).
-       */
-      heapUsed: number,
-      /**
-       * External memory (bytes).
-       */
-      external: number,
-    },
-    /**
-     * CPU usage percentage.
-     */
-    cpu: number,
-    /**
-     * Map containing garbage collection stats.
-     */
-    gc: Map<GCOp, GCAggregatedEntry>
-    /**
-     * Number of active handles
-     */
-    activeHandles: number,
-    /**
-     * Object containing raw values
+declare interface MemoryMetric {
+   /**
+    * RSS memory (bytes).
+    */
+   rss: number,
+   /**
+    * Total heap Memory (bytes).
+    */
+   heapTotal: number,
+   /**
+    * Heap memory used (bytes).
+    */
+   heapUsed: number,
+   /**
+    * External memory (bytes).
+    */
+   external: number,
+   /**
+     * arrayBuffers memory (bytes), it's not presetn on Node 10
```

Comment:
typo: `presetn`

Reply:
Thinking about this, I'd probably prefer it to be worded a little differently.. Maybe:

```js
arrayBuffers memory (bytes). This value is `undefined` on Node <= 10.
```

or something a little more explicit.